packages/python2-plaso/PKGBUILD

```diff
@@ -0,0 +1,52 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python2-plaso
```

Comment:
Please, try to build bot python and python2 packages when uploading it, we are trying to avoid the major number of python2 packages since that python3 is available.

Reply:
Multiple deps are still only with python2 available.
Making python3 packages is now planned as TODO ...

Reply:
Thanks, please answer the couple of questions that I made there and modify the PKGBUILD in order to build the python3 package, I will merge it then.

Reply:
Will do once PR https://github.com/BlackArch/blackarch/pull/2273 is merged and fine.

Reply:
It was merged :)